### ISSUE TITLE
In accordion I cannot set all items to be closed by default.

### ISSUE THREAD

**Issue by algdev-gr** [yiisoft/yii2-bootstrap4#219]
### What steps will reproduce the problem?
Create an accordion widget with any number of groups.

### What's expected?
All accordion groups to be closed by default.

### What do you get instead?
First group is always rendered showing not matter what options I set.

### Additional info

| Q                | A
| ---------------- | ---
| Yii vesion       | Latest
| PHP version      | 8.0
| Operating system | Windows 10

Hi everyone,

I am trying to created an accordion widget that by default has all it's groups not showing, but instead the first group is always showing by default.

I tried reading the source code from accordion.php it seems that the BC fix is enforcing this behavior without any option to configure it.

Did I miss anything?
Appreciate your input.
Regards

**Comment by simialbi** [comment]
What about setting
```php
[
    'items' => [
        [
            // [...],
            'expand' => false
        ]
    ]
]
```

**Comment by algsupport** [comment]
@simialbi thank you for your response. I believe I have tried it.
I will check again and update you, but I would like to get your attention to these lines of code:

https://github.com/yiisoft/yii2-bootstrap4/blob/a0280a6caefbf153ab49664c9ef129675dd02040/src/Accordion.php#L138

from line 138 to line 146 you will find this code:

```
$expanded = array_search(true, ArrayHelper::getColumn(ArrayHelper::toArray($this->items), 'expand', true));
        foreach ($this->items as $key => $item) {
            if (!is_array($item)) {
                $item = ['content' => $item];
            }
            // BC compatibility: expand first item if none is expanded
            if ($expanded === false && $index === 0) {
                $item['expand'] = true;
            }
```

If I am not mistaken, it means that if none of the items have expand set to true, then set expand to true for the first item/group.
Please correct me if I am wrong.

Thank you for your time
Regards

**Comment by simialbi** [comment]
@algsupport you're right, it's done because of compatibility with [yii\bootstrap\Collapse](https://github.com/yiisoft/yii2-bootstrap/blob/master/src/Collapse.php)

**Comment by algsupport** [comment]
@simialbi I got that. But I think there should be a way to explicitly disable that behavior.

Regards
